### DISCUSSION TITLE
[NGINX-LUA] support custom operation ignore

### DISCUSSION THREAD

**Discussion by wangrzneu** [apache/skywalking#8086] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

1.support operation ignore according to some custom configs like:
. ignore: /test,/heathCheck/
. ignorePrefix: /css/,/html/
. ignoreSuffix: .css,.html

2.in reverse proxy case, the sw8 header will not be transmitted to the upstream for ignored operation

### Use case

1.support custom operation ignore
We do not need to record all request like some static resources. With this feature, we can ignore some kind of requests.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
`Ignore mechanism` is fine, as Java has this already. Meanwhile, the sampling is recommended to do in the backend, especially rate policy. Because once you set up sampling at agent side, the metrics, even maybe topology, are not correct any more in the whole SkyWalking.

**Comment by wu-sheng** [comment]
Another alternative and recommend way is setting different routine rules to separate tracing and no tracing request. 
In this case, you would be more cautious and have better performance due to native path mapping mechanism.

**Comment by wu-sheng** [comment]
Moving to discussion if further discussion is required.
